documentation/Troubleshooting.md

```diff
@@ -1,43 +1,43 @@
 This section provides troubleshooting information for KubeMarine and Kubernetes solutions.
 
 - [KubeMarine Errors](#kubemarine-errors)
-  - [KME0001: Unexpected exception](#kme0001-unexpected-exception)
```

Comment:
@shmo1218  pls revert KME000* doc changes. Since their names should exact match with source code. For example: `KME0001: Unexpected exception`

Reply:
Hi, @koryaga,

I have reverted the changes done for KME000#. Please take a look.